cernopendata/modules/fixtures/data/articles/vms/md/alice.md

```diff
@@ -0,0 +1,88 @@
+# ALICE Virtual Machines: How to install
+
+1.  [1\. How to install VirtualBox](#vbox)
+2.  [2\. Create a VM](#create)
+3.  [3\. Configure a VM](#configure)
+
+
+## Step 1: Installing VirtualBox
+
+VirtualBox is a free, open source and multiplatform application to run virtual machines: you can [download](https://www.VirtualBox.org/wiki/Downloads) the package for your platform from the Downloads page.
+
+You will need administrative privileges ("root" privileges) on every platform to perform the installation of VirtualBox.
+
+Note: the latest tested version of VirtualBox working with CernVM is 4.3.14\. If you have troubles with the latest verion of VirtualBox, pick that one: the full history of VirtualBox versions is available [on a different page.](https://www.VirtualBox.org/wiki/Download_Old_Builds)
+
+
+## Step 2: Creating a Virtual Machine
+
+
+Go on the [CernVM downloads page](http://cernvm.cern.ch/portal/downloads) and download the appropriate version: you must pick the OVA x86_64 for VirtualBox.
+
+<img src="http://opendata.cern.ch/img/alice/1.png" width="50%"><br>
```

Comment:
The images won't be necessarily on this URL after we update the site. We need to add them as part of this commit.
We can do it by adding them to static assets.

(and we should do it for `cernopendata/modules/fixtures/data/articles/getting_started/md/cms.md` as well...)

Reply:
ok! 👍🏻